server/views/pages/edit-goal.njk

```diff
@@ -0,0 +1,213 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "../components/error-summary/error-summary.njk" import errorSummary %}
+{% from "hmpps/components/date-picker/macro.njk" import hmppsDatePicker %}
+{% from "moj/components/side-navigation/macro.njk" import mojSideNavigation %}
+
+{% set pageTitle %}
+    {% if errors %}
+        Error: {{ applicationName }} - Edit a goal with {{ data.popData.givenName }}
```

Comment:
Should these say "Change a goal" ?

Reply:
Ok it should be change a goal, and I Think I modified that... checking where its lost now